handbook/upgrade/upgrade.md

```diff
@@ -0,0 +1,280 @@
+
+1. Provision a new Window server for SystemLink.
+
+1. Install and configure the new version of SystemLink Server.
+
+1. Install `nislmigrate` on your new SystemLink Server.
+
+1. Attach the `D:\` volume used to capture data from your original SystemLink Server.
+
+1. Run the command `nislmigrate restore --all --secret <your secret> --dir D:\migration`.
+
+1. Verify your new SystemLink Server has all the expected migrated data.
+
+## Single Node to Multi Node Migration
+
+This section describes workflows used when you intend to upgrade or migrate from a single node SystemLink Server configuration to a multi node SystemLink Server configuration that makes use of dedicated servers for MongoDB, PostgreSQL, or file storage. While not required, these migration workflows include provisioning a new SystemLink Server. This is done to reduce risk and ensure your original SystemLink Server is always in an operable state.
+
+### Single Node to Multi Node with MongoDB
+
+This workflow describes steps to upgrade a single node deployment of SystemLink Server to a multi node deployment where the MongoDB instance used by SystemLink is hosted on its own server or replica set.
+
+1. Backup your SystemLink Server.
+
+1. Attach a volume to store data captured by `nislmigrate`. This will be referenced as `D:\`.
+
+1. Install `nislmigrate`.
+
+1. Run the command `nislmigrate capture --all --secret <your secret> --dir D:\migration`.
+
+1. Detach `D:\`.
+
+    !!!note
+        At this point in the workflow your original SystemLink Server is in an operable state. Be aware if any new data is created or ingested by SystemLink Server at this time it will not be available to your new server.
+
+1. Provision a new Windows server for SystemLink.
+
+1. Provision a new MongoDB server or replica set.
+
+1. Install and configure the same or a newer version of SystemLink Server.
+
+1. Configure SystemLink to use the newly created MongoDB server or replica set.
+
+1. Install `nislmigrate` on your new SystemLink Server.
+
+1. Attach the `D:\` volume used to capture data from your original SystemLink server.
+
+1. Run the command `nislmigrate restore --all --secret <your secret> --dir D:\migration`.
+
+1. Verify your new SystemLink Server has all the expected migrated data.
+
+### Single Node to Multi Node with File Storage
+
+This workflow describes steps to upgrade a single node deployment of SystemLink Server to a multi node deployment where the file storage instance used by SystemLink is hosted on a a dedicated NAS, SAN, or AWS S3.
+
+1. Backup your SystemLink Server.
+
+1. Attach a volume to store data captured by `nislmigrate`. This will be referenced as `D:\`.
+
+1. Install `nislmigrate`.
+
+1. Run the command `nislmigrate capture --all --secret <your secret> --dir D:\migration`.
+
+1. Copy the files from the original file storage location to the new file store.
```

Comment:
Copy the files from the original file storage location to the new file store using one of the following options.

Table with migration location and method here.

Reply:
Done. Let me know if the table covers your guidance as expected.